src/integrations/third-party/jetpack.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package Yoast\WP\SEO\Integrations\Third_Party
+ */
+
+namespace Yoast\WP\SEO\Integrations\Third_Party;
+
+use Yoast\WP\SEO\Conditionals\Front_End_Conditional;
+use Yoast\WP\SEO\Conditionals\JetPack_Conditional;
```

Comment:
There is a difference in naming: the class is defined as `Jetpack_Conditional`, but is imported  and used here as `JetPack_Conditional` (notice the capital P).

Reply:
Oh right, I assumed it was a 'capital P dangit' case 😉 But then I saw it was not so I renamed. Forgot the import. Good catch!